maploc/data/dataset.py

```diff
@@ -181,8 +212,13 @@ def get_view(self, idx, scene, seq, name, seed, bbox_tile):
 
         if self.cfg.return_gps:
             gps = self.data["gps_position"][idx][:2].numpy()
-            xy_gps = self.tile_managers[scene].projection.project(gps)
-            data["uv_gps"] = torch.from_numpy(canvas.to_uv(xy_gps)).float()
+            world_t_gps = self.tile_managers[scene].projection.project(gps)
+            world_t_gps = torch.from_numpy(world_t_gps).float()
+            world_T_gps = Transform2D.from_Rt(torch.eye(2), world_t_gps)
```

Comment:
Since the GPS never has any orientation, it would make more sense to keep `world_t_gps` only.